metricflow/specs/specs.py

```diff
@@ -594,6 +594,37 @@ def linkable_specs(self) -> LinkableSpecSet:  # noqa: D
             entity_specs=self.entity_specs,
         )
 
+    @override
+    def merge(self, other: MetricFlowQuerySpec) -> MetricFlowQuerySpec:  # noqa: D
+        # Can't merge if both properties are set.
```

Comment:
Wait, why can we merge when these don't match at all?

Can you please add a docstring explaining this?